data/json/monsters/turrets.json

```diff
@@ -220,5 +220,55 @@
     "death_drops": {  },
     "death_function": [ "BROKEN" ],
     "flags": [ "SEES", "NOHEAD", "ELECTRONIC", "IMMOBILE", "NO_BREATHE", "DROPS_AMMO" ]
+  },
+  {
+    "id": "mon_turret_riot",
```

Comment:
I think we need to stick very close to reality with any new robots we add and I can't find any systems like this in use in 2019. There's [this thing](https://www.army-technology.com/contractors/unmanned_vehicles/technorobot/) and maybe the SWORDS system if that 6 barrel launcher takes 40x46mm grenades but I can't see any sign of widespread deployment for these systems.